x-pack/plugins/security_solution/public/detections/components/rules/step_define_rule/index.tsx

```diff
@@ -511,10 +514,54 @@ const StepDefineRuleComponent: FC<StepDefineRuleProps> = ({
       />
     );
   }, [kibanaDataViews]);
+
+  const tourConfig = {
+    currentTourStep: 1,
+    isTourActive: true,
+    tourPopoverWidth: 300,
+    tourSubtitle: i18n.DATA_SOURCE_GUIDE_SUB_TITLE,
+  };
+
+  const [tourState, setTourState] = useState(() => {
```

Comment:
`tourState` is currently typed as `any` 😦

Reply:
I couldn't find much documentation on giving a function to `useState`, but it only seems to be invoked once. Is this what you expect? What happens if that data isn't in localstorage on initial invocation?

Reply:
also should be resolved here: 992d397dcf49356d421dce1dbcb4fec84336b543